### DISCUSSION TITLE
Syncing files from other apps

### DISCUSSION THREAD

**Discussion by SimonPickup** [MobiusSync/MobiusSync#102] [Show and tell]
As of v1.23.1, we now have support for syncing files outside the Möbius Sync sandbox, i.e. you can sync files owned by other apps in their sandbox.

This is marked EXPERIMENTAL for now, but is available to the public. Of course, make sure you have backups.

Since Syncthing is modifying the files in a sandbox managed by another app, there is a risk it can be confused, worst case resulting in data corruption or data loss.

This discussion thread has been created to gather reports of successes (and failures?) with different app's data.

**Comment by SimonPickup** [comment]
I have successfully synced **VLC** media.

**Comment by SimonPickup** [comment]
I have successfully synced **Acrobat** PDF files.

**Comment by dragisha** [comment]
I had success sharing GoodReader sandbox but not on the first try. Maybe some debug log would come in handy for people trying this feature?

**Reply by tessaherself** [reply]
How did you solve it? I tried 10times already

**Comment by wc8** [comment]
Is it possible to sync two sandboxed folders, aka an iPadOS app folder and an iPhoneOS app folder? 
Sync of Obsidian from first iOS to mac seemed to go well, but, perhaps for user error, the sync to 2nd iOS landed in MobiusSync folder instead of Obsidian folder.

**Reply by wc8** [reply]
It seems to be working, early stages. I have each iOS Obsidian folder (sandbox) pointed to the _**same**_ macOS folder. I have staggered versioning on.

**Reply by tyagank** [reply]
I think there is  bug at least in the iOS version, even when I select the folder explicitly through the "Pick External Folder" it still ends up pointing to the MobiusSync folder.

**Reply by berot3** [reply]
> I think there is  bug

Is this still a problem?

**Reply by cuboooooo** [reply]
Similar issue, whether I create the synced folder first on my iPhone or on my PC, the resulting files still get placed into the Mobius Sync sandbox, not the Obsidian one. I got it to work once, but I could not repeat those results. I found a solution here: https://github.com/MobiusSync/MobiusSync/discussions/102#discussioncomment-6115520 

Edit: Here are the steps to get it to work, I was able to repeat this a couple times.
1. Create a Vault on your PC, and put a note in it for confirmation once you sync. (Alternitavely, copy all your existing Vault Data into this one, or choose your pre-existing vault(didn't work with mine for some odd reason))
2. Share that folder to your phone in Syncthing
3. When accepting the folder in Mobius, pick external folder, navigate to Obsidian, and create a new folder with the same name as the FOLDER LABEL IN SYNCTHING you're sharing.
4. Go to Obsidian app, the vault should automatically be in your collection, and enjoy!

Important: the only way I found it to be successful is if both folders, vaults, and both folder labels within Syncthing and Mobius all have the same name.

**Reply by Simontage** [reply]
Worked！

**Reply by dieterbaier** [reply]
Small addition: for me, this only worked if I had not created the folder to be synchronised as a subfolder. For example, I could only synchronise the Obsidian folder 'syncthingTest' if it was created directly in ~/ on the Mac. It did not work as a subfolder of e.g. 'obsidian'. But I have not verified it. At least my first attempt was ~/obsidian/syncthingTest (not successful) and the second attempt ~/syncthingTest (was successful).

**Reply by serhii-freeman** [reply]
Hi guys @dieterbaier @Simontage @MasterCubo @tyagank , do you sync any media files (images, video, audio) in your vault?
Does it synchronize settings and plugins as well? (I assume not)
Thanks.

**Reply by dieterbaier** [reply]
I do sync the whole vault. Even the settings. So far it works fine. The only thing you should take care of is not having open Obsidian on different devices at the same time. If I forget this approach it leads to sync comflicts some times.
For that, I will try something else if I get time: having all my templates, journal notes (mostly dataview Scripts), scripts, the Obsidian folder itself etc. in git (since they don't change offen) and the real notes within a subfolder which is gitignored. With syncthing I would Sync everything except the Obsidian folder.
Following this approach, my Obsidian setup would become a kind of template (e.g. to setup a vault on the customers side, where I want to Use my setup but not all notes) and my 'private' notes would be synced via syncthing 'on the fly' through my devices. 
I know that I will not have the current status of my workspace on all devices but that's OK for me.
If I change something in my setup (e.g. installing a new plugin), I would use git for updating my vaults (need to find out what to gitignore from the Obsidian folder).

**Reply by serhii-freeman** [reply]
Thank you @dieterbaier for your quick response!

**Comment by kodfodrasz** [comment]
Does this mean that the Photos/Videos can be synced (backed up from) an iPhone using MobiusSync once this feature is released??

**Reply by SimonPickup** [reply]
Apple Photos Library is a separate thing, not file-based, so this is NOT covered at this stage. However you can sync other photos/videos as files and use a third-party app to display them (e.g. VLC).

**Reply by SimonPickup** [reply]
For reference, the enhancement requests are #4 and #9.

**Reply by dumbuz** [reply]
Likely you can sync between Mobius Sync's sandbox and Photos using Shortcuts. Not sure exactly how you'd set this up, but it's probably very possible.

**Comment by tyagank** [comment]
Versions
Windows 11 - v1.23.2, Windows (64-bit Intel/AMD)
iOS 16 and the latest MobiusSync version from Appstore

The sync is not working for any app folders. Only works for the MobiusSync Folder. I even paid the $4.99 it asked for before letting me select an external folder. Very annoying.

**Reply by SimonPickup** [reply]
The way iOS works is when opening the file picker, it defaults to the Möbius folder, but you can navigate out of there to other app sandboxes.

**Reply by kodfodrasz** [reply]
I have also had problems understanding the UI workflows, and while I understand this is because of external factors, I believe probably some videos on some really difficult workflows would be cool in the documentation.

**Reply by var1amov** [reply]
Same here. It _could_ sync with app folder just once, but it resets to MobiusSync app folder soon. I am testing with Obsidian app (markdown text editor). I am realize, that "use different app folders" is just a beta version of app features, but since user able to test it only after pay 4.99$, it is frustrating. Hope you will fix it and iOS users could have syncthing client on their devices!

**Reply by tessaherself** [reply]
yes, and then it saves as the file path: "bookmark:obsidian" or whatever... but it ends up creating a new folder "obsidian" in the möbius-sync sandbox

**Reply by tessaherself** [reply]
@var1amov  did you succeed in the end or did they offer refund?

**Reply by var1amov** [reply]
@TimoWacke no, I didn't want to refund. Moreover, it just start working with Obsidian and I am happy now.

**Reply by tessaherself** [reply]
@var1amov  how did you do that? :O any tipps?

**Reply by var1amov** [reply]
It just works as desired. 
1. I've shared folder on second device
2. When I got notification about new share at iOS I've added it and choose Obsidian app folder
3. Profit

**Comment by nchilada** [comment]
Thank you! This works with Keepassium if I select the top-level (`Keepassium`) folder. I assume it would work with StrongBox and other KeePass implementations, too.

A couple other notes:

* Although Background App Refresh seems to help out at times, it's obviously necessary to open the Möbius Sync app if you want to guarantee an immediate sync.
* I got an "Error: Unsupported folder" whenever I tried to select an _empty_ external folder; fortunately this was avoidable by adding a placeholder file to whatever folder I wanted to sync.

**Reply by MatthewIIZ** [reply]
I experience an issue with Keepassium whereby the app is installed but no app sandbox is created in Files. All efforts to create a folder and add a placeholder file in it result in Mobius Sync producing the error “unsupported folder”.
Would you have any advice on resolving this?

**Comment by wiivanil** [comment]
I have successfully synced Obsidian.

**Reply by phlepper** [reply]
I was able to do this as well (on iPad)

My steps:
1. I created a vault on iPad with Obsidian (empty vault)
2. I shared my Obsidian Vault folder (from Linux) to my iPad
3. In Mobius, I selected "Pick External Folder" -> "On My iPad" -> Obsidian **and then created a subfolder with the same name as the folder I was sharing (Obsidian Vault)**, then clicked on "Open" and then "Save" back in Mobius.

Note that if I just chose Obsidian and didn't create the subfolder, Mobius wouldn't sync and I got an error "Error on folder 'xyz' at bookmark:abc: folder marker missing...")

**Reply by pookhaosc1** [reply]
I followed the same steps but weren't able to replicate. All the sync files ended up in the Mobius's sandbox folder and not Obsidian's.

**Reply by githubarooski** [reply]
Any further luck on this? My experience is the same as @pookhaosc

**Reply by berot3** [reply]
I thought about buying pro, but since there seem quite a couple not succeeding I might as well wait until it is more stable ...

**Reply by tessaherself** [reply]
@wiivanil how? I am struggling

**Comment by lucendio** [comment]
I have trouble to *Pick External Folder*. When I tap the button, the font gets brighter but the file picker does not open. I'm on latest iOS (16.5) and latest App version (1.23.2). Any ideas what I might hold wrong? Is it a paid feature? Coincidentally, there is also nothing happening, when I tap the *Purchase* item in main menu.

**Comment by Tokarak** [comment]
IOS 15; Palera1n jailbreak. Can't get it to work with Logseq. 
It appears it just isn't writing anything to the folder. When I create the files on my laptop and try to get it to sync _into_ my phone, it claims the files are synced, but I can't find anything in files.app. Maybe I'll have a look later using an unsandboxed file explorer. But first I'll try this comment:

> 1. I created a vault on iPad with Obsidian (empty vault)
> 2. I shared my Obsidian Vault folder (from Linux) to my iPad
> 3. In Mobius, I selected "Pick External Folder" -> "On My iPad" -> Obsidian and then created a subfolder with the same 
 name as the folder I was sharing (Obsidian Vault), then clicked on "Open" and then "Save" back in Mobius.

**Reply by Tokarak** [reply]
Yes, I can confirm that the guide worked. Official docs should be updated since it's non-obvious that the external folder shoul exist (that is, it's not created automatically).
Also, what hasn't been explained by this is why I couldn't share an existing folder (what I managed is to get a folder shared to me). Maybe the file system permissions don't allow access to subdirectories which weren't created by Mobious? That would be odd design, but I can't think of any other reason.

**Comment by MrOggy85** [comment]
I made these 2 tests:
* Add "Logseq" folder directly under "On My iPhone" folder . This resulted in creating a folder in the "Möbius Sync" folder
  * `On My iPhone/Logseq` **Failure** 🙅
* Use Logseq to create a _new graph_ (results in a new folder under "Logseq" folder). Then I added this new subfolder. This resulted in "Möbius Sync" using the folder I wanted.
  * `On My iPhone/Logseq/my_graph` **Success** 🙆 

My _suspicion/theory_ is that this new experimental feature does not work for "root" folders directly in "On My iPhone". So you always need to target a subfolder for the external app you want to sync. Maybe this has been explained, but it was not clear to me when using the app.

**Reply by jorge-gbs** [reply]
This worked for me.

**Reply by PCjabber** [reply]
The second method described has been working for me with Logseq iOS (iPad) for the past ~18 months. 
FWIW, I’m using the freemium (in-app purchase) version of Möbius Sync, and syncing to several other Syncthing nodes (Linux 1.28.x, Windows 1.30.x, Android 1.27.x) over LAN, WAN, and Relay WAN.

**Reply by SkyLeite** [reply]
I'm syncing the root folder and it works just fine. No setup needed.

**Comment by jcotton42** [comment]
Has anyone tried this with the iOS version of Scrivener?

**Reply by dictionarymouse** [reply]
My experience:
Scrivener won't read it's subfolders except iPad/Scrivener/Tutorial. 
Möbius won't sync folders in the iPad directory, (e.g. iPad/Scrivener). 
Attempting to sync iPad/Scrivener with a folder on Mac/Scrivener, Möbius will start syncing mac folder(s) with any iPad/MöbiusSync folder(s)...

**Comment by tessaherself** [comment]
So how did it work out eventually? Any tips?

**Comment by tessaherself** [comment]
many thanks!

after a lot of tries I got mine to work too

**Reply by serhii-freeman** [reply]
Do you have any media files (images, video, audio) in your vault?
Does it synchronize settings and plugins as well? (I assume not)
Thanks.

@SimonPickup you really can increase your income from the app if you enable limited pro option for test purposes. For example 1 Mb storage maximum. I am currently not sure if it works, it is good to have an ability to try first.

**Comment by dictionarymouse** [comment]
Currently I am only able to sync if

1. the folder is a **subfolder** of an app folder (e.g. iPad/app/sync-folder
2. the iOS folder to be shared **exists** and is **empty** when I attempt to share it 
3. an app subfolder sync is started (shared) **from iOS** 16 (tested on iPadOS), <br> which means `iPad/app/subfolder` can't be shared with `iPhone/app/subfolder`, as far as I can tell. 
It seems an iOS-device/app/subfolder can be shared with any computer folder and any iOS MobiusSync subfolder

**Reply by dictionarymouse** [reply]
I spoke too soon... interesting results, though.
I had added `iPad/Obsidian/empty-new-folder` _(or so I thought)_ but what iPad/Obsidian/ is what sync'd to the macOS folder.
So I tried adding iPad/Scrivener/subfolder, and the Open Folder button in Möbius Sync now opens `iPad/Scrivener`.

⚠️ 💡 _Note: in my experience, whatever iPad folder opens when you tap `Open Folder`in MöbiusSync is what will sync_ 
I **strongly** recommend testing newly added folders before syncing with another device, especially if already sharing folders.

**Comment by dictionarymouse** [comment]
### Tips for External Folder Sync

- ❗ After adding or editing a folder, test folder selection by **Open Folder** before syncing 
-  ⏸️ pausing SyncThing on your other devices until the folders have been added/labeled on the iOS device might help the folder selection stick
- ❗ Changing the Folder Label after sharing an external app _might_ default the folder to iPad/MobiusSync
- 💡 [Cannot-Share-Previously-Shared-Folder workaround](https://github.com/MobiusSync/MobiusSync/issues/111#issuecomment-1510146763)
- ❔ [Possible folder selection workaround](https://github.com/MobiusSync/MobiusSync/issues/123#issuecomment-1773577151)
- 💾 Make Backups!

**Comment by amzon-ex** [comment]
Hi! This happens to be the only use-case I plan to use Möbius for; and I'm willing to try out the feature. However, the fact that it is both experimental (people seem to have had issues with it) and paywalled (albeit a reasonable amount) keeps me from doing it. Would I get a refund if I pay, try and it doesn't work out?

**Reply by bass000** [reply]
+1 or add a free tier (20MB?) for external folders or time (7days). So we can try on our own app (report here)!

Not willing to pay if it doesn't work.

**Comment by MatthewIIZ** [comment]
Currently experiencing an issue with v1.23.2 using the Premium build. Obsidian syncing ok. Keepassium sync failing during setup.
Keepassium not generating a sandbox folder for the application. Creating a folder “On My iPad/KeePassium” and selecting this folder results in the error “Unsupported Folder”. Tested with and without placeholder file and folder.

**Comment by bass000** [comment]
Has anybody tried goodnotes?

**Comment by mugshepherd** [comment]
I'm having issues after running through the steps at [above](https://github.com/MobiusSync/MobiusSync/discussions/102#discussioncomment-7344237).  (iphone SE gen3 and Mbp M1.  

I'm noticing a few things things (Trusted/Untrusted devices; Root/Non-root folder; Edits on one device not able to sync to another).  The first two seem seem related to each other, and the third which I think is a separate issue:  

1. in situations where both devices are trusted, the external folder I choose is ignored and it syncs a folder in Möbius Sync with the alias name if I provided one.  If i did not provide one it syncs the parent Möbius sync folder as the shared folder name (ie: in the format of e.g.: zwxyg-idwrr) - i then get all sub folders of Möbius Sync on my computer under that folder name.  

2. when both devices are untrusted (or trusted, but shared and encrypted) and the shared password for each device is added when sharing from the other:
    2.a if the folder is a root folder (e.g.: `a-shell/` Möbius sync throws errors*
    2.b if the folder is a nested folder (e.g.: `a-shell/test/`) then it is indeed synced to my computer.  
        2.b.1 (and really the third issue I mentioned):  After files are synced between the two devices, I can edit them on the Mbp and they sync to the iPhone, but if I edit them on the iPhone it fails to sync to my Mbp, with the error, `syncing: finishing: pull: generic error`.  And then the syncing is broken.  

2.b.1 happens when I sync both internal and external folders, which leads me to believe it's a configuration issue. But I've tried 1. adding Syncthing to permitted apps in my Mbp Firewall permissions; 2. specifically opening up port 22000 for both TCP and UDP traffic (as recommended via the docs at https://docs.syncthing.net/users/firewall.html), and 3. even turning off my firewall as well.  Both phone and computer are clean installs of Syncthing (via Syncthing-Mac and Möbius Sync - i had mucked around with the first install and realized something I did might have been causing this, so uninstalled and reinstalled on both devices) and I can't get my computer to sync after editing a file on my phone regardless of what I do.  Have looked online and the syncthing forums (e.g [this issue](https://forum.syncthing.net/t/untrusted-device-doesnt-finish-syncing-when-file-last-modified-by-unknown-or-old-device/17270) but so far have not found anything that has worked.  Will post an issue there regarding this last issue.  


\* logs for the error on trying to share a root folder here (and note that I did provide the encryption password when I synced the folder)
```
2024-01-14 21:48:18: Failed to create folder root directory mkdir /private/var/mobile/Containers/Data/Application/612C91B2-CDFE-4889-86A4-97EF581D80A5/ashell: operation not permitted
2024-01-14 21:48:18: Error on folder "ashell" (sykqy-idvff) at ~/ashell: folder path missing
2024-01-14 21:48:19: Failure checking encryption consistency with device mbp16 for folder "ashell" (sykqy-idvff): open /private/var/mobile/Containers/Data/Application/612C91B2-CDFE-4889-86A4-97EF581D80A5/ashell/.stfolder/syncthing-encryption_password_token: no such file or directory
2024-01-14 21:48:20: Failure checking encryption consistency with device mbp16 for folder "ashell" (sykqy-idvff): open /private/var/mobile/Containers/Data/Application/612C91B2-CDFE-4889-86A4-97EF581D80A5/ashell/.stfolder/syncthing-encryption_password_token: no such file or directory
2024-01-14 21:48:22: Failure checking encryption consistency with device mbp16 for folder "ashell" (sykqy-idvff): open /private/var/mobile/Containers/Data/Application/612C91B2-CDFE-4889-86A4-97EF581D80A5/ashell/.stfolder/syncthing-encryption_password_token: no such file or directory
```

**Comment by cbmiller2610** [comment]
I was able to successfully get Obsidian sync working using Mobius Sync. However, there is a quirk. Here are the successful steps:

1. Create an Obsidian Vault subfolder on Linux within a parent folder (you will sync the parent folder)
2. Install syncthing on Linux
3. Add the parent folder as a sync folder on the Linux syncthing server, making sure the folder path is correct.
4. Install Mobius Sync, and Install Obsidian on Iphone
5. Create an Obsidian Vault on IOS (this will be extra, and unused, it is just to make the Obsidian sandbox folder appear in your local files)
6. Retrieve the device ID of the Iphone from Mobius sync (upper right settings cog, show ID)
7. Use the device ID to add the Iphone as a remote device in the Linux syncthing server. When adding, go to the "Sharing" tab and check the sync folder you set up earlier as shared.
8. Retrieve the device ID of the Linux computer, and use it to add a remote device to Mobius sync.  No need to check any folders, unless you have others from the phone you want to share to the Linux computer.
9. You will get a notification in the Mobius app that the Linux computer wants to share a folder. Accept the notification, and the folder dialogue will pop up. 
10. In the folder dialogue, go to the Folder Path and below it select "Pick External Folder".  Navigate to the Obsidian sandbox folder in your phone's local files, and hit "Open" on the top right (do not descend into the subfolder of the Obsidian vault you made earlier). This will select the Obsidian sandbox folder as the parent sync folder on the Iphone.
11. Check the "Local State (Total)" in the Mobius app, it should now show the correct file count reflecting what was just shared.  Go to the Files app and verify that the folder with your Obsidian Vault from Linux has been synced to the Obsidian sandbox folder.

The above works. However, when I tried to add the actual Obsidian Vault folder as the sync folder on Linux, and give it the same name as the Vault folder on Iphone (as someone suggested), it did not work. I got errors like the following: 
LMZP4] 2024/01/16 22:28:25 INFO: Established secure connection to VVZPLY6 at 10.0.0.130:22000-10.0.0.33:22000/tcp-client/TLS1.3-TLS_CHACHA20_POLY1305_SHA256/LAN-P10-2ULG9FU0DSPOE7GK6KD0FEVM3C
[LMZP4] 2024/01/16 22:28:25 INFO: Device VVZPLY6 client is "syncthing v1.23.2-41-ga6a9adeeb-i" named "localhost" at 10.0.0.130:22000-10.0.0.33:22000/tcp-client/TLS1.3-TLS_CHACHA20_POLY1305_SHA256/LAN-P10-2ULG9FU0DSPOE7GK6KD0FEVM3C
**[LMZP4] 2024/01/16 22:28:26 INFO: Unexpected folder "Zettelkasten" (ma5mg-pug6c) sent from device "VVZPLY6-Z6P2XPI-DC4FW7V-GWW5WV3-S33ICOG-XAMWTMY-V54KV3Q-FKI54Q2"; ensure that the folder exists and that this device is selected under "Share With" in the folder configuration.**
[LMZP4] 2024/01/16 22:28:40 INFO: Adding folder "Zettelkasten" (ma5mg-pug6c)
[LMZP4] 2024/01/16 22:28:40 INFO: No stored folder metadata for "ma5mg-pug6c"; recalculating
[LMZP4] 2024/01/16 22:28:40 INFO: Ready to synchronize "Zettelkasten" (ma5mg-pug6c) (sendreceive)
[LMZP4] 2024/01/16 22:28:40 INFO: Device VVZPLY6 folder "Zettelkasten" (ma5mg-pug6c) has a new index ID (0xAFEB4581C67DC9EF)
[LMZP4] 2024/01/16 22:28:40 INFO: Completed initial scan of sendreceive folder "Zettelkasten" (ma5mg-pug6c)

This caused the initial file sync to fail (silently, it seems), and no files were synced from the Linux computer to the Iphone.  So it seems that Parent folders with different names on Linux and Iphone might be required for the sync directory, and this seems to allow subfolders (Obsidian Vaults) with the same name on both computers to sync correctly.

**Comment by SimonPickup** [comment]
A nice write-up and discussion re Obsidian here: https://forum.obsidian.md/t/sync-mac-pc-and-ios-using-syncthing-mobius-sync/72022

**Reply by connordear** [reply]
I was able to get this working with Obsidian, but the instructions above were a little vague to integrate the iPad into an existing sharing network. I have it successfully synced between PC/Android/iPad now!

### On iPad
1. Install Moebius Sync Pro
2. Install Obsidian
3. Create a tmp vault to create the Obsidian sandbox folder (the "Obsidian" folder under "On My iPad)
4. From Moebius Sync, click "+ Add Folder", and ensure you use the "Pick External Folder" to go to the "on My iPad/Obsidian" directory, and then create the folder you want to be your vault (I named it the same as my vault on Android/PC)
5. Share that folder to your other devices.

### On PC/Android
7. **When accepting the share request _point the folder path (where this "new share" should go) on your PC/Android to the existing vault you wish to share_**. I think if you didn't have an existing vault it could go anywhere.
8. In my case, since I was already syncing between PC/Android, I paused the previous share/folder between them, and now they all share across the iPad sync. (having both sharing running ended up in a bunch of conflicts)

Some of the other comments indicate they were able to share an existing folder to the iPad, then use the "Pick External Folder" to drop it into the Obsidian Sandbox, but this did not work for me, I had to start the share from iPad.

**Comment by lucendio** [comment]
Following up on [my own post](https://github.com/MobiusSync/MobiusSync/discussions/102#discussioncomment-6146247), speaking for iOS only, it's indeed possible to sync folders outside of the sandbox of the app, but it requires the __pro__ (paid) version. This should be communicated in a more visible manner.

**Reply by bfunc** [reply]
> Following up on [my own post](https://github.com/MobiusSync/MobiusSync/discussions/102#discussioncomment-6146247), speaking for iOS only, it's indeed possible to sync folders outside of the sandbox of the app, but it requires the **pro** (paid) version. This should be communicated in a more visible manner.

When I click on "Pick External Folder" wild In-app purchase pops up and clicking on "Continue Free" does nothing.. It is ok for me to pay 6 euro, but it is very confusing, +1  to "This should be communicated in a more visible manner".

**Comment by mountainmohawk** [comment]
Is there any risk with RetroArch? I am only looking at syncing my save files.

**Comment by Moominologist** [comment]
I can successfully sync large folders (1000+ files, nested structure) from my mac to a folder freshly created by Moebius sync. What's kind of irritating is that *all* the synced subfolder have the modfication time of the first sync. All files, however, have their proper modfication time as on the source (mac). Even more bewildering: If the same folder is synced into the Moebius sandbox, *all timestamps (folders and files)* are correct. 
Is this something that should be reported here or should I open a separate issue about this, @SimonPickup ?
